src/FlatpakRefFile.vala

```diff
@@ -88,6 +101,31 @@ public class Sideload.FlatpakRefFile : Object {
         }
     }
 
+    public async string? get_name () {
+        // Application name from AppData is preferred
+        if (appdata_name != null) {
+            return appdata_name;
+        }
+
+        if (!yield load_key_file ()) {
+            return null;
+        }
+
+        string? name = null;
+
+        // Otherwise, does the .flatpakref have a Title field?
+        // This is supposed to be the human readable name of the app suitable for display
+        // But is sometimes still the app ID
+        try {
+            name = key_file.get_string (REF_GROUP, "Title");
```

Comment:
This `name` string is not being returned from this `get_name ()` method, is that intentional?